.github/workflows/patch-release-build.yml

```diff
@@ -99,6 +99,67 @@ jobs:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
           GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Create checksum
+        run: |
+          echo ORIG_CHECKSUM="$(shasum otelagent/build/libs/aws-opentelemetry-agent-*.jar | awk '{ print $1 }')" >> "$GITHUB_ENV"
+
+      - name: Get current version
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        shell: bash
+        run: |
+          echo ADOT_JAVA_VERSION="$(./gradlew printVersion -q )" >> $GITHUB_ENV
```

Comment:
to be aligned with the release of the adot java jar, we need to use version `${{ github.event.inputs.version }}` can you change that?